alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/parcel/parcel.component.html

```diff
@@ -90,12 +132,6 @@ <h4 [id]="parcel.uuid">Parcel #{{ parcelInd + 1 }}: Parcel and Owner Information
         </div>
         <div>{{ owner.phoneNumber }}</div>
         <div>{{ owner.email }}</div>
-        <div *ngIf="parcel.ownershipTypeCode !== PARCEL_OWNERSHIP_TYPES.CROWN">
-          <a *ngIf="owner.corporateSummary" (click)="onOpenFile(owner.corporateSummary)">{{
-            owner.corporateSummary.fileName
-          }}</a>
-          <app-no-data *ngIf="!owner.corporateSummary"></app-no-data>
```

Comment:
Same thing as application regarding keeping the anchor link while removing the condition